docs/configuration.md

```diff
@@ -97,6 +97,18 @@ storage:
 		poolname: radospool
 		username: radosuser
 		chunksize: 4194304
+	swift:
+		username: username
+		password: password
+		authurl: https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth
```

Comment:
@snayakie thx for ur information. If you can access ur storage provider only with username and password/apikey, you should use the keystone v1.
@lebauce correct me if I am wrong.